lighthouse-core/lib/page-functions.js

```diff
@@ -354,7 +360,7 @@ function getNodeSelector(node) {
 function isPositionFixed(element) {
   /**
    * @param {HTMLElement} element
-   * @param {string} attr
+   * @param {'overflowY'|'position'} attr
```

Comment:
this was just easier since `CSSStyleDeclaration` doesn't have a string index, and `CSSStyleDeclaration[keyof CSSStyleDeclaration]` includes numbers and null. `getStyleAttrValue` is an inner convenience function, and it's unlikely to ever see any more values here (but more could always be added, of course) so this seems fine